src/wsk-tasks/babel.js

```diff
@@ -20,25 +20,60 @@
 'use strict';
 
 const gulp = require('gulp');
-const babel = require('gulp-babel');
 const uglify = require('gulp-uglify');
-const rollup = require('gulp-rollup');
 const sourcemaps = require('gulp-sourcemaps');
+const browserify = require('browserify');
+const buffer = require('vinyl-buffer');
+const source = require('vinyl-source-stream');
+const path = require('path');
+const glob = require('glob');
 
-function build() {
-  let stream = gulp.src(GLOBAL.config.src + '/**/*.js')
-    .pipe(sourcemaps.init())
-    .pipe(rollup())
-    .pipe(babel({
-      presets: ['es2015']
-    }));
+function bundleJS(fullFilePath) {
+  const browserifyBundles = browserify({
+    entries: fullFilePath
+  });
+
+  // Rollupify reduces the size of the final output but increases build
+  // time to do it so enable for production build only
+  if (GLOBAL.config.env === 'prod') {
+    browserifyBundles.transform('rollupify');
+  }
+
+  let stream = browserifyBundles
+  .transform('babelify', {presets: ['es2015']})
+  .bundle()
+  // `source` Converts Browserify's Node Stream to a Gulp Stream
+  // Use path.relative to make the file have the correct home in `dest`
+  .pipe(
+    source(path.join('.', path.relative(GLOBAL.config.src, fullFilePath)))
+  )
+  .pipe(buffer())
+  .pipe(sourcemaps.init());
```

Comment:
Shouldn’t sourcemaps be init()’d before rollup/browserify/whatever come into play? Or do they do they own sourcemap thing?